PsfRuntime/Config.cpp

```diff
@@ -255,11 +255,58 @@ void LogCountedStringW(const char* name, const wchar_t* value, std::size_t lengt
 
 static const psf::json_object* g_CurrentExeConfig = nullptr;
 
+// Note: Recursive
+FILE* find_json(std::filesystem::path basefolder)
+{
+    FILE* fRet = NULL;
+    WIN32_FIND_DATA FindFileData;
+    HANDLE hFind;
+
+#if _DEBUG
+    Log("Searching for config in folder: %ls", basefolder.c_str());
+#endif
+
+#pragma warning(suppress:4996) // Nonsense warning; _wfopen is perfectly safe
+    fRet = _wfopen((basefolder / L"config.json").c_str(), L"rb, ccs=UTF-8");
+    if (fRet)
+    {
+        Log("Found config at %ls", (basefolder / L"config.json").c_str());
+        return fRet;
+    }
+    // If not found, enumerate subdirectories recursively until found.

```

Comment:
nite: Line break.

Reply:
I don't understand what the ask is, but the function was removed anyway.